packages/cli/package.json

```diff
@@ -12,7 +12,6 @@
   },
   "bugs": "https://github.com/boostercloud/booster/issues",
   "dependencies": {
-    "@boostercloud/framework-core": "^0.11.5",
```

Comment:
This is great!
![](https://i.kym-cdn.com/photos/images/newsfeed/000/591/926/700.jpg)